test/flycheck-test.el

```diff
@@ -3885,6 +3885,13 @@ Why not:
      '(5 9 warning "Redundant braces after class definition"
          :checker scala-scalastyle))))
 
+(flycheck-ert-def-checker-test scheme-chicken scheme nil
+  (skip-unless (funcall (flycheck-checker-get 'chicken-scheme 'predicate)))
```

Comment:
I added Geiser to Cask. How do I enable `geiser-mode` in the test? `make integ` is still skipping the test at the moment.